amazon_emr_serverless_image_cli/assets/image-manifest.yaml

```diff
@@ -1,5 +1,89 @@
 !ImageManifest
 EmrReleases:
+- !EmrRelease
+  ReleaseName: emr-6.12.0
+  Images:
+  - !ImageDetail
+    ImageType: spark
+    EnvironmentVariable:
+    - java_home
+    - spark_home
+    FileStructure:
+    - bin-files
+    - hadoop-files
+    - hadoop-jars-6.10
+    - java-bin
+    - spark-bin
+    - spark-jars-6.12
+    ManifestConfig: !ManifestConfig
+      Entrypoint: /usr/bin/entrypoint.sh
+      User: hadoop:hadoop
+      WorkingDir: /home/hadoop
+  - !ImageDetail
+    ImageType: hive
+    EnvironmentVariable:
+      - hadoop_home
+      - hadoop_libexec_dir
+      - hadoop_user_home
+      - hadoop_yarn_home
+      - hive_home
+      - java_home
+      - tez_home
+      - yarn_home
+    FileStructure:
+      - hadoop-files
+      - hadoop-jars-6.10
+      - hadoop-yarn-jars
+      - hive-bin-files
+      - hive-jars-6.12
+      - java-bin
+      - tez-jars
+    ManifestConfig: !ManifestConfig
+      Entrypoint: /usr/bin/entrypoint.sh
+      User: hadoop:hadoop
+      WorkingDir: ""
+- !EmrRelease
+  ReleaseName: emr-6.11.0
+  Images:
+  - !ImageDetail
+    ImageType: spark
+    EnvironmentVariable:
+    - java_home
+    - spark_home
+    FileStructure:
+    - bin-files
+    - hadoop-files
+    - hadoop-jars-6.10
```

Comment:
should this hadoop jar version here be 6.11?

Reply:
6.11 and 6.12 has the same files as 6.10, so using the same config. It's called 6.10 to distinguish from 6.9, which had different files.